apps/nextjs/src/app/aila/[id]/download/DownloadView.tsx

```diff
@@ -0,0 +1,176 @@
+"use client";
+
+import { AilaPersistedChat } from "@oakai/aila/src/protocol/schema";
+import { Box, Flex, Grid } from "@radix-ui/themes";
+
+import Layout from "@/components/AppComponents/Layout";
+import Button from "@/components/Button";
+import DialogContents from "@/components/DialogControl/DialogContents";
+import { DialogRoot } from "@/components/DialogControl/DialogRoot";
+import { DownloadButton } from "@/components/DownloadButton";
+import { Icon } from "@/components/Icon";
+
+import { SurveyDialogLauncher } from "./SurveyDialogLauncher";
+import { useDownloadView } from "./useDownloadView";
+
+type DownloadViewProps = Readonly<{
+  chat: AilaPersistedChat;
+  featureFlag: boolean;
+}>;
+export function DownloadView({
+  chat,
+  featureFlag,
+}: Readonly<DownloadViewProps>) {
+  const { lessonPlan } = chat;
+  const {
+    lessonSlidesExport,
+    worksheetExport,
+    lessonPlanExport,
+    starterQuizExport,
+    exitQuizExport,
+    additionalMaterialsExport,
+    sections,
+    totalSections,
+    totalSectionsComplete,
+  } = useDownloadView(chat);
```

Comment:
This diff is mostly changing the file name. But the introduction of this hook is the substantive change in this file